test/authentication.js

```diff
@@ -161,7 +161,7 @@ describe('authentication', function () {
 				assert(body);
 
 				request({
-					url: nconf.get('url') + '/api/me',
+					url: nconf.get('url') + '/api/user/admin',
```

Comment:
this test should stay as /api/me and return the admin users info

Reply:
`/api/me` isn't redirected to `/api/user/admin` because it uses the redirect helper because AJAX redirects don't work. I could make it so `/api/me` returns the JSON that `/api/user/[userslug]` and _also_ the redirect header that ajaxify depends on, or I could just undo the removal of the `/me` handler.

Do certain parts of core depend on `/me` working as it does now?

Reply:
I dont think core depends on /api/me returning the json data but some plugins/themes might so either we add it back or its going to be a breaking

Reply:
..change